NeoMathEngine/src/CPU/CpuMathEngine.cpp

```diff
@@ -241,17 +241,20 @@ void CCpuMathEngine::GetMathEngineInfo( CMathEngineInfo& info ) const
 	info.AvailableMemory = SIZE_MAX;
 }
 
-IPerformanceCounters* CCpuMathEngine::CreatePerformanceCounters() const
-{
 #if FINE_PLATFORM( FINE_ANDROID ) || FINE_PLATFORM( FINE_LINUX )
+IPerformanceCounters* CCpuMathEngine::CreatePerformanceCounters( bool isOnlyTime ) const {
+	if ( isOnlyTime )
```

Comment:
_codestyle_ **always** use figure brackets in `if` statements (even if there's only one line)